snitun/client/connector.py

```diff
@@ -105,7 +105,7 @@ async def start(
                 end_host,
                 end_port,
             )
-            await multiplexer.delete_channel(channel)
+            multiplexer.delete_channel(channel)
```

Comment:
I'm a bit rusty on my Python async handling, what happens if this throws an exception now that it'll run outside the try/except here?

Reply:
It will be an unhandled exception the coro ends and the [asyncio exception handler](https://superfastpython.com/asyncio-event-loop-exception-handler/) will likely log 

Its no longer possible for `delete_channel` to raise MultiplexerTransportError so if it does raise here we likely have a bug